packages/docs-site/src/library/pages/components/list.md

```diff
@@ -0,0 +1,113 @@
+---
+title: List
+description: todo
+header: true
+---
```

Comment:
Worth adding `draft: true` to the frontmatter if the design docs are due to come at a later date.